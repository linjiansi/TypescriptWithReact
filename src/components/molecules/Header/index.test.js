import React from 'react';
import renderer from 'react-test-renderer';
import { HeaderComponent } from './index';

it('renders correctly at HeaderComponent', () => {
  const tree = renderer
    .create(<HeaderComponent headerType='main'></HeaderComponent>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
