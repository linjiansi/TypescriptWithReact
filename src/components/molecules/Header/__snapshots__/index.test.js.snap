// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly at HeaderComponent 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-header-3 MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root makeStyles-title-2 MuiTypography-h6"
      >
        BookManager
      </h6>
      <a
        href="/login"
        onClick={[Function]}
      >
        <div
          className="makeStyles-main-5"
        >
          <button
            className="MuiButtonBase-root-34 MuiButton-root-7 MuiButton-contained-15 makeStyles-button-6 MuiButton-containedSecondary-17"
            disabled={false}
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label-8"
            >
              LOGOUT
            </span>
          </button>
        </div>
      </a>
    </div>
  </header>
</div>
`;
